audio_to_spectrogram/README.md

```diff
@@ -157,3 +157,16 @@ roslaunch audio_to_spectrogram sample_audio_to_spectrogram.launch
     - `~spectrum` (`jsk_recognition_msgs/Spectrum`)
 
       Spectrum data calculated from audio by FFT.
+
+  - ### Parameters
+    - `~plot_amp_min` (`Double`, default: `0.0`)
+
+      Minimum value of amplitude in plot
+
+    - `~plot_amp_max` (`Double`, default: `20.0`)
+
+      Maximum value of amplitude in plot
+
+    - `~queue_size` (`Int`, default: `1000`)
```

Comment:
What about lowering the default value

Reply:
> What about lowering the default value

@708yamaguchi
Is it OK to you?
Maybe you set this value as `1000`:
https://github.com/jsk-ros-pkg/jsk_recognition/commit/06ed79c5b3867fc811be7ea4a664a61ff54f8ea9#diff-463547bd0daff5521153fdd38a2ef6ee98bc441134e20073f047c70853c57072R28

Reply:
I am sorry that I do not remember clearly, but the reason to increase queue_size was perhaps my unstable network environment.

If the spectrum comes in late due to poor communication conditions, some spectrum may be missing if the queue size is small.
Only one spectrum is missing and the spectrogram is not output correctly.
So I increased queue size to receive the spectrum without missing out.

But I don't have an idea about the appropriate queue size. It depends on user's environment. (I feel 1000 is too big.)

Reply:
Since this node is specialized for visualizing spectrum, the default queue_size can be small.
Even if the queue is large, msg will be discarded if it exceeds the queue size.

Reply:
OK, I see.
Thank you for your explanation.

Reply:
@708yamaguchi So you mean you don't remember why you set `1000`?

Reply:
As far as I remember, at that time, I didn't want the topic missing out.
But the queue_size does not have big effect as iory pointed out.

Reply:
Thank you. I set default queue_size 1.